samples/command_executer/README.md

```diff
@@ -24,7 +24,7 @@ remove yet).
 and build an EIF.
 
 ```
-	$ export NITRO_CLI_BLOBS=$(realpath ../../blobs/)
+	$ export NITRO_CLI_BLOBS=$(realpath ../../blobs/x86_64)
 	$ nitro-cli build-enclave --docker-dir "./resources" --docker-uri mytag --output-file command-executer.eif
 ```
 ---
```

Comment:
Can you also update the output of the step 4, to be similar to the one in https://docs.aws.amazon.com/enclaves/latest/user/cmd-nitro-run-enclave.html#cmd-nitro-run-enclave-example. It has a different format in this README. For example:

```
Start allocating memory...
Started enclave with enclave-cid: 16, memory: 2048 MiB, cpu-ids: [1, 5, 2, 6]
{
    "EnclaveID": "i-abc12345def67890a-enc9876abcd543210ef12",
    "ProcessID": 12345,
    "EnclaveCID": 16,
    "NumberOfCPUs": 4,
    "CPUIDs": [
        1,
        5,
        2,
        6
    ],
    "MemoryMiB": 2048
}
```